release/air_tests/air_benchmarks/workloads/train_benchmark.py

```diff
@@ -0,0 +1,163 @@
+import ray
+from ray.air import session
+from ray.train.torch import TorchTrainer
+from ray.air.config import ScalingConfig
+
+import time
+import os
+import json
+
+import torch
+import torchvision
+
+DEFAULT_IMAGE_SIZE = 224
+
+# This benchmark does the following:
+# 1) Read images with ray.data.read_images()
+# 2) Apply preprocessing with map_batches()
+# 3) Train TorchTrainer on processed data
+# The resulting metrics are:
+# - ray.data+transform: Throughput of data ingest (steps 1+2 above)
+# - ray.torchtrainer.fit: Throughput of the final epoch in
+#   TorchTrainer.fit() (step 3 above)
+
+
+def iterate(dataset, label, metrics):
```

Comment:
let's reuse the same method in https://github.com/ray-project/ray/blob/master/release/nightly_tests/dataset/image_loader_microbenchmark.py#L17-L27, we can put the method into a util.py.